lang/rust/crates/litex-pac/src/constants.rs

```diff
@@ -1,10 +1,10 @@
-/* automatically generated by rust-bindgen 0.69.1 */
+/* automatically generated by rust-bindgen 0.60.1 */
 
 pub const CONFIG_CLOCK_FREQUENCY: u32 = 57120000;
 pub const CONFIG_CPU_RESET_ADDR: u32 = 0;
 pub const CONFIG_CPU_COUNT: u32 = 1;
-pub const CONFIG_CPU_ISA: &[u8; 15] = b"rv32i2p0_mafdc\0";
-pub const CONFIG_CPU_MMU: &[u8; 5] = b"sv32\0";
+pub const CONFIG_CPU_ISA: &[u8; 15usize] = b"rv32i2p0_mafdc\0";
```

Comment:
I know you didn't do this, but these look like syntax errors. Things still built fine?

Reply:
Could be related to our installed versions of quartus?

Reply:
Why would this be related to Quartus? The original constants file is generated as part of the LiteX Python steps, and this file is generated by the `litex-pac` generation `make` script, and has nothing to do with anything Quartus.